### PR TITLE
Added placeholders in update contact form and open a ticket

### LINKED ISSUE #2737: Interactive messages- Automatic wait for response node (Result)
**Is your feature request related to a problem? Please describe.**
Result tab in wait for response which is automatically created from interactive message is hidden (as save as). Also, there is no automatic result name that gets populated, like in regular wait for response. Due to this, the chances to orgs missing to add a result variable is higher (QA recently missed adding result name for one of their important questions)
image
image

**Describe the solution you'd like**
We need to update the design for the same (similar to regular wait for response node) and have an default result name added to avoid this. 

**Describe alternatives you've considered**

**Additional context**

@mdshamoon

### LINKED ISSUE #2811: Default label/Name shown should be changed or removed
**Describe the task**
In the below instances default name/label is shown, which will create confusion/ might be misleading

1) Poetry in "Ticketing system". It can be replaced by the text "Create/ Select"
image

2) "Language" in Update Contact. It should be replaced by "Select"
image

### PR DESCRIPTION
**Pull request by akanshaaa19**
Target issues:
- https://github.com/glific/glific-frontend/issues/2811
- https://github.com/glific/glific-frontend/issues/2737

### CODE REVIEW

**Comment by kurund** [comment]
@akanshaaa19 

I am wondering if this is worth submitting to upstream?

@mdshamoon 
Thoughts?

File: src/components/flow/actions/sendinteractivemsg/SendInteractiveMsgForm.tsx [hunk 1]
```diff
@@ -140,12 +140,13 @@ export default class SendMsgForm extends React.Component<
...
       valid = true;
     }
-
     if (valid) {
       this.props.updateAction(stateToAction(this.props.nodeSettings, this.state));
       if (this.props.nodeSettings.originalNode.ghost) {
         this.props.resetNodeEditing();
-        this.props.updateRouter(stateToRouter(this.props.nodeSettings, this.state));
+        this.props.updateRouter(
+          stateToRouter(this.props.nodeSettings, this.props.assetStore, this.state)
```
Comment (kurund) [inline src/components/flow/actions/sendinteractivemsg/SendInteractiveMsgForm.tsx]: we are changing the signature of the function, I would prefer adding new params at the end.

File: src/components/flow/actions/sendinteractivemsg/helpers.tsx [hunk 2]
```diff
@@ -195,7 +195,7 @@ export const stateToRouter = (
     valid: true
   };
 
-  const renderedNode = stateToNode(settings, result);
+  const renderedNode = stateToNode(settings, assetStore, result);
```
Comment (kurund) [inline src/components/flow/actions/sendinteractivemsg/helpers.tsx]: see previous comment.

File: src/components/flow/actions/updatecontact/UpdateContactForm.tsx [hunk 3]
```diff
@@ -309,6 +309,7 @@ export default class UpdateContactForm extends React.Component<
         <p>{i18n.t('forms.select_what_to_update', 'Select what to update')}</p>
         <AssetSelector
           name={i18n.t('forms.contact_field', 'Contact Field')}
+          placeholder="Select"
```
Comment (kurund) [inline src/components/flow/actions/updatecontact/UpdateContactForm.tsx]: this needs to be localized.

File: src/components/flow/routers/ticket/TicketRouterForm.tsx [hunk 4]
```diff
@@ -199,6 +199,7 @@ export default class TicketRouterForm extends React.Component<
             <TembaSelect
               key="select_topic"
               name={i18n.t('forms.topic', 'Topic')}
+              placeholder="Select"
```
Comment (kurund) [inline src/components/flow/routers/ticket/TicketRouterForm.tsx]: needs to localized.

**Review by kurund — CHANGES_REQUESTED** [review]

**Review by mdshamoon — APPROVED** [review]

**Pull request merged**